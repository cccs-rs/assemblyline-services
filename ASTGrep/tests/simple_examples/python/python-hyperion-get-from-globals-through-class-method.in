class Square:
    def _power(Invert = bool):
        return globals()[Invert]

Square._power(Invert='LLLLJJLLLILJIJJJIIJIII')
