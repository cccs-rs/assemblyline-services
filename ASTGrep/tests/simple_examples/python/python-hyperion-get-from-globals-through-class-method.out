class Square:
    def _power(Invert = bool):
        return globals()[Invert]

LLLLJJLLLILJIJJJIIJIII
